check-cla/action.yml

```
@@ -102,6 +102,34 @@ runs:
               name: '${{ inputs.label }}'
             })
 
+    # checkout conda/infrastructure to update .clabot
+    - uses: actions/checkout@v3
+      if: steps.contributors.outputs.hasSigned == 'false'
+      with:
+        repository: conda/infrastructure
+
+    # update .clabot
+    - shell: python
+      if: steps.contributors.outputs.hasSigned == 'false'
+      run: |
+        import json
+        from pathlib import Path
+
+        path = Path(".clabot")
+        clabot = json.loads(path.read_text())
+        clabot["contributors"].append("${{ github.actor }}")
```

The list is currently sorted. If we want to keep that, a sort is required after the append:
```suggestion
        clabot["contributors"].append("${{ github.actor }}")
        clabot["contributors"].sort()
```